File: Problem Statements/Mensuration/Circle and Sphere/Circumference/CoC.java

//Circumference of a Circle
import java.util.Scanner;
public class CoC{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter Radius : ");
        int r=sc.nextInt();
        sc.close();
        double circum=2*Math.PI*r;
        System.out.println("Circumference : "+circum);
    }
}

File: Problem Statements/Mensuration/Circle and Sphere/Circumference/CoC.cpp
//Circumference of a Circle
#include<iostream>
#include<math.h>
using namespace std;
#define Pi 3.14

void main(){
    int r;
    float circum=0.0;
    cout<<"Enter the Radius : ";
    cin>>r;
    circum = 2*Pi*r;
    cout<<"Circumference : "<<circum;
}